x-pack/plugins/index_management/public/services/navigation.js

```diff
@@ -12,5 +12,8 @@ export const getUrlService = () => {
   return urlService;
 };
 export const getFilteredIndicesUri = (filter) => {
-  return encodeURI(`#${BASE_PATH}indices/filter/${encodeURIComponent(filter)}`);
```

Comment:
If we want to change the semantics like this, we need to change the name of this function.  Maybe getIndexListUri?  Also, we don't need to call encodeURI if there's not a filter, as the URI is already browser safe.

Reply:
renamed to getIndexListUri() and removed encodeURI. thanks!